src/main.cpp

```diff
@@ -4112,13 +4282,13 @@ bool ReceivedBlockTransactions(
     CAmount saplingValue = 0;
     CAmount orchardValue = 0;
     for (auto tx : block.vtx) {
-        // Negative valueBalance "takes" money from the transparent value pool
-        // and adds it to the Sapling value pool. Positive valueBalance "gives"
+        // Negative valueBalanceSapling "takes" money from the transparent value pool
+        // and adds it to the Sapling value pool. Positive valueBalanceSapling "gives"
         // money to the transparent value pool, removing from the Sapling value
         // pool. So we invert the sign here.
-        saplingValue += -tx.valueBalance;
+        saplingValue += -tx.GetValueBalanceSapling();
 
-        // Orchard valueBalance behaves the same way as Sapling valueBalance.
+        // Orchard valueBalanceSapling behaves the same way as Sapling valueBalanceSapling.
```

Comment:
Regex mismatch, but we can clean this up further:
```suggestion
        // valueBalanceOrchard behaves the same way as valueBalanceSapling.
```